opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/operations/RequisitionImport.java

```diff
@@ -73,4 +75,7 @@ public boolean isAborted() {
         return false;
     }
 
+    public static boolean isValidRequisitionImport(RequisitionImport ri) {
+        return ofNullable(ri).isPresent() && ofNullable(ri.getRequisition()).isPresent();
+    }
```

Comment:
This is misleading. For validity of a requisition, there is more than just checking for `null`.
In addition, what's wrong with a plain `ri != null`?

Reply:
Well true, but since we are using in this project at least java 11, i think we can use some of its features; I can of course rename the method to isRequisionImportPresent

Reply:
I don't got why this was refactored into a separate method at the first place.

Reply:
Ok, let me just remove all the refactoring things, at the end that could be a separate task